packages/openshift/util.py

```diff
@@ -146,4 +146,32 @@ def mkdir_p(path):
             pass
         else:
             raise
-    return path
\ No newline at end of file
+    return path
+
+# unit scale used by kubernetes
+unit_scales = {'n':-3, 'u':-2, 'm':-1, 'k':1, 'K':1, 'M':2, 'G':3, 'T':4, 'P':5, 'E':6}
+def extract_numerical_value(val):
```

Comment:
Please add unit tests. e.g. tools/openshift-client-python/packages/openshift/test_selector.py is a test for selectors. All unit tests can be executed with tools/openshift-client-python/run_unittests.sh.